src/network/my_name_message_handler_sub_system.cpp

```diff
@@ -128,8 +129,27 @@ void MyNameMessageHandlerSubSystem::Execute( Message const& message )
         lifecycleMsg->mState = LifecycleMessage::SoldierProperties;
         lifecycleMsg->mClientId = msg.mSenderId;
         mMessageHolder.AddOutgoingMessage( std::auto_ptr<Message>( lifecycleMsg.release() ) );
+
         EventServer<engine::ConnectionEvent>::Get().SendEvent( engine::ConnectionEvent( msg.mSenderId, engine::ConnectionEvent::Connected ) );
     }
+    // calculate checksums and distribute it to the clients
+    {
+        static boost::uint32_t datapkgChecksum = PackageChecksum("data.pkg");
+        static boost::uint32_t autoidChecksum = FileChecksum("autoids");
```

Comment:
there is one under run/data. I used that for testing (by copying next to the executable)
did I misunderstand the requirement again and autoid checking should be done otherwise?
or autoids is part of data.pkg?